react/features/notifications/components/native/NotificationsContainer.js

```diff
@@ -72,6 +71,15 @@ class NotificationsContainer extends Component<Props> {
     componentDidMount() {
         // Set the initial dismiss timeout (if any)
         this._manageDismissTimeout();
+
+        Animated.timing(
```

Comment:
Let's put the animations in the individual notifications, so they can all fade in.